docs/adr-002-ebrelayer-eth-subscribe.md

```diff
@@ -0,0 +1,42 @@
+# ADR 002: Ebrelayer Ethereum Subscribe
+
+## Changelog
+
+- 2020/10/21: Initial version
+
+## Status
+
+*Proposed*
+
+## Context
+In this ADR, we discuss the solution for ebrelayer how to subscribe the events from Ethereum and process these events.
+### Summary
+
+For ebrelaye, it just need subscribe the events from CosmosBridge smart contract, both LogLock event and LogBurn event, then process both events and send transaction to Sifchain. Basically, there are two problems.
```

Comment:
Overall, this is good docs, some minor changes, I do believe it is subscribed to the bridgebank, not the cosoms bridge, and `ebrelaye` should turn be `ebrelayer`